p4proto/p4src/linux_networking/linux_networking.p4

```diff
@@ -322,9 +322,8 @@ control linux_networking_control(inout headers_t hdr,
             hdr.outer_ipv4.dst_addr : exact @name("ipv4_dst");
         }
         actions = {
-            drop;
             @tableonly decap_outer_ipv4;
-            @defaultonly set_exception;
```

Comment:
Should we remove defined set_exception action as well since it's not used?
   action set_exception(PortId_t vport) {
        send_to_port(vport);
        local_metadata.exception_packet = 1;
    }

Reply:
This PR is already merged and closed....!!!

There has been a discussion on this and definitions are left alone, and it can be re-used when we implement exception path and miss action.